src/main/java/Deadline.java

```diff
@@ -0,0 +1,16 @@
+public class Deadline extends Task{
```

Comment:
There should be a space after ```Task``` and ```{``` right? I noticed the same issue in other places, so just mind that :)